### PR TITLE
Typo fixes in README

### PR DESCRIPTION
**Pull request by brettdewoody**

### CODE REVIEW

**Comment by gliechtenstein** [comment]
Thanks! Looking through this i found another typo haha, you missed it too :) 

https://github.com/intercellular/cell/commit/3848952da11997183c799d6aacce3ce12bfa691d

File: README.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ To use Cell, you simply define a variable that describes the DOM content and beh
 
 ## Rule #1. Attributes map 1:1 to DOM attributes by default.
 
-When you define a javascript object, its attributes map 1:1 to DOM attributes. So,
+When you define a Javascript object, its attributes map 1:1 to DOM attributes. So,
```
Comment (devsnek) [inline README.md]: its actually `JavaScript`

File: README.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ maps to:
 
 <br>
 
-## Rule #2. Use 7 special keywords to declare the cell structrure
+## Rule #2. Use 7 special keywords to declare the cell structure
```
Comment (devsnek) [inline README.md]: should be a colon here

File: README.md [hunk 3]
```diff
@@ -238,7 +238,7 @@ When it finds one, it takes that blueprint object (called a `"Genotype"` in Cell
 
 So far this is just a static DOM tree. To make it dynamic, you need to write a program that "remote controls" these HTML elements.
 
-Normally javascript frameworks maintain a separate **centralized data structure and application context ([Model-View-Controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) or some variation)** that synchronizes with and controls HTML elements dynamically.
+Normally Javascript frameworks maintain a separate **centralized data structure and application context ([Model-View-Controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) or some variation)** that synchronizes with and controls HTML elements dynamically.
```
Comment (devsnek) [inline README.md]: JavaScript

File: README.md [hunk 4]
```diff
@@ -274,7 +274,7 @@ Comparison 	|Frameworks before Cell		| Cell
 -------------|--------------|------------------------------
 Control 		| Centralized 		| Decentralized
 Structure		| A master Model-View-Controller program that controls all the HTML elements | Each html element as the container of its own Model-View-Controller logic
-Your App		| Full of framework API syntax | Just a vanillla javascript. No framework code.
+Your App		| Full of framework API syntax | Just a vanillla Javascript. No framework code.
```
Comment (devsnek) [inline README.md]: JavaScript, and you missed the vanilla with three l's

File: README.md [hunk 5]
```diff
@@ -296,7 +296,7 @@ Here are some of the reasons why these middlemen have been necessary, and **why
 > Most web app frameworks depend on other complex libraries (Don't forget to `npm install` before doing anything!) Cell doesn't depend on any library.
 
 ##### 3. Frameworks introduce dependencies.
-> Just by choosing to use a framework you have already lost the war against dependency. From then on, you need to use `npm install` for every frontend javascript library you need to use. Cell frees you from this loop and lets you use frontend Javascript libraries with simple `<script src>`.
+> Just by choosing to use a framework you have already lost the war against dependency. From then on, you need to use `npm install` for every frontend Javascript library you need to use. Cell frees you from this loop and lets you use frontend Javascript libraries with simple `<script src>`.
```
Comment (devsnek) [inline README.md]: "war against dependency" and JavaScript

**Pull request merged**